src/id128/id128.c

```diff
@@ -316,6 +324,29 @@ static int run(int argc, char *argv[]) {
         if (r <= 0)
                 return r;
 
+        if (arg_var_partition_uuid) {
+                /* The DPS says that the UUID for /var/ should be keyed with machine-id. */
+                _cleanup_(table_unrefp) Table *table = NULL;
+                sd_id128_t uuid;
+
+                if (!sd_id128_is_null(arg_app))
+                        return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
+                                               "'show --app-specific=' can only be used with explicit UUID input.");
+
+                r = sd_id128_get_machine_app_specific(SD_GPT_VAR, &uuid);
+                if (r < 0)
+                        return log_error_errno(r, "Failed to generate machine-specific UUID: %m");
+
+                r = show_one(&table, "var-uuid", uuid, /* first= */ false);
```

Comment:
This should mimic verb_machine_id and verb_invocation_id BTW - the input is fixed, and there's only one output, so no need for show_one()